vulnerable_people_form/form_pages/shared/routing.py

```diff
@@ -298,8 +298,12 @@ def get_redirect_for_returning_user_based_on_tier():
         latest_location_tier = location_eligibility.get_uprn_tier(original_uprn)
         latest_shielding_advice = location_eligibility.get_shielding_advice_by_uprn(original_uprn, current_app)
```

Comment:
Would there need to be an override for uprn as well in the test data?

Reply:
Don't think that's needed. The uprns for the test addresses are stored as NULL in the db. `originial_uprn` should be `None` in this function and shouldn't go into that if block.